### ISSUE TITLE
Comments for functions

### ISSUE THREAD

**Issue by SplashDance** [small-yellow-duck/kaggle_art#1]
Hey Ducky, 

The scripts you provided are really helpful. However, they'd be even _more_ helpful if you added some documentation to your functions (according to, for instance, the [Google Style Guide](https://google.github.io/styleguide/pyguide.html)) to make it easier for **R** users like me to learn from your code.

Thanks!
Steve

**Comment by small-yellow-duck** [comment]
Yep, I completely agree.... :-) Writing documentation for the siamese
network script is about fourth on the list of priorities for me at the
moment. It does help my motivation to know that people are actually looking
at the code, so thanks for your note!

**Comment by SplashDance** [comment]
Hey, no worries--just throwing that out there. Take your time.

Thanks again for your work--it's a lot to take in!
